lib/ui/notifications.dart

```diff
@@ -16,48 +17,182 @@ GlobalConfiguration cfg = GlobalConfiguration();
 
 class NotificationsWidgetState extends State<NotificationsWidget> {
   GlobalConfiguration cfg = GlobalConfiguration();
+  var notificationSubList = <NotificationSubscription>[];
+  var notificationSub = new NotificationSubscription('Test Keyword');
+  //var notificationsList = <Notification>[];
+  //var notification = new Notification(DateTime.now(), '', '', '');
+
 
   @override
   void initState() {
     super.initState();
   }
```

Comment:
You don't need to override this, since it's not doing anything different than the original implementation.

Reply:
You can remove the entire function, rather than just the override annotation.